<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab2360">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>991</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab2400">
     <attribute name="title">
      <string>2400</string>
     </attribute>
     <widget class="QWidget" name="mplwindow" native="true">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>-30</y>
        <width>721</width>
        <height>421</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="mplvl">
       <item>
        <widget class="QWidget" name="widget_2" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pauseButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>380</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 128, 0);</string>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>441</y>
        <width>781</width>
        <height>211</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Time Stamp</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>X</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Y</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="sweepButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>380</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 0);</string>
      </property>
      <property name="text">
       <string>Sweep</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>380</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);
font: 75 14pt &quot;Helvetica&quot;;</string>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="paraGroup">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>241</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>231</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="paraLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </layout>
      </widget>
      <widget class="QLabel" name="modeLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Helvetica&quot;;
border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
      <widget class="QComboBox" name="modeComboBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>151</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Voltage Bias</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Voltage Sweep</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Current Sweep</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Current Bias</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Voltage Customized</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Current Customized</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QPushButton" name="saveButton">
      <property name="geometry">
       <rect>
        <x>809</x>
        <y>451</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Data</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>2360</string>
     </attribute>
     <widget class="QWidget" name="mplwindow_2" native="true">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>451</width>
        <height>401</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="mplvl_2">
       <item>
        <widget class="QWidget" name="widget_3" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pauseButton_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>450</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="configLabel">
    <property name="geometry">
     <rect>
      <x>-435</x>
      <y>473</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="startGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="scanButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>10</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>121</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="instruLayout"/>
    </widget>
    <widget class="QPushButton" name="configButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>14</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 0);
alternate-background-color: rgb(179, 179, 179);
</string>
     </property>
     <property name="text">
      <string>Assign</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 0);</string>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>131</width>
      <height>192</height>
     </rect>
    </property>
    <property name="movement">
     <enum>QListView::Free</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
